libsolidity/interface/OptimiserSettings.h

```diff
@@ -59,7 +59,7 @@ struct OptimiserSettings
 		"]"
 		"jmul[jul] VcTOcul jmul";      // Make source short and pretty
 
-	static char constexpr DefaultYulOptimiserCleanupSteps[] = "fDnTOc";
+	static char constexpr DefaultYulOptimiserCleanupSteps[] = "fDnTOcm";
```

Comment:
We should probably make it ``mu`` in the end to get rid of the dead variables, right?

Reply:
Sure, I'll add it in a separate commit so we can see the diff with `u` and without.